lib/Db/Message.php

```diff
@@ -331,6 +359,8 @@ static function (Tag $tag) {
 			'summary' => $this->getSummary(),
 			'encrypted' => ($this->isEncrypted() === true),
 			'mentionsMe' => $this->getMentionsMe(),
+			'avatar' => $this->avatar ? $this->avatar->jsonSerialize() : null,
```

Comment:
```suggestion
			'avatar' => $this->avatar?->jsonSerialize(),
```

https://php.watch/versions/8.0/null-safe-operator